problems/strings.js

```diff
@@ -103,15 +153,44 @@ function disemvowel() {}
  * @returns {string}
  */
 
- function disemvowelLongWords(){}
+const disemvowelLongWords = (str) => {
```

Comment:
This functions still not passing the specs. Please revisit. Think about ways to use the other diemvowel function you wrote as a helper.